File: LeaveRequest.java
import java.util.Date;

public class LeaveRequest {
    private int id;
    private int securityStaffId; // Referencing SecurityStaff
    private Date startDate;
    private Date endDate;
    private String reason;
    private String status; // Pending, Approved, Declined

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getSecurityStaffId() {
        return securityStaffId;
    }

    public void setSecurityStaffId(int securityStaffId) {
        this.securityStaffId = securityStaffId;
    }

    public Date getStartDate() {
        return startDate;
    }

    public void setStartDate(Date startDate) {
        this.startDate = startDate;
    }

    public Date getEndDate() {
        return endDate;
    }

    public void setEndDate(Date endDate) {
        this.endDate = endDate;
    }

    public String getReason() {
        return reason;
    }

    public void setReason(String reason) {
        this.reason = reason;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: StaffsManagementSys_winform/StaffsManagementSys_winform/Models/LeaveRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace StaffsManagementSys_winform.Models
{
    public class LeaveRequest
    {
        public int Id { get; set; }
        public int SecurityStaffId { get; set; } // Reference to SecurityStaff
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Reason { get; set; }
        public string Status { get; set; } // Pending, Approved, Declined
    }
}
